save-plugins/warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugin/warn/WarnPluginConfig.kt

```diff
@@ -35,24 +35,13 @@ data class WarnPluginConfig(
     val execCmd: String,
     val warningsInputPattern: Regex,
     val warningsOutputPattern: Regex,
-    val warningTextHasLine: Boolean = true,
-    val warningTextHasColumn: Boolean = true,
+    val warningTextHasLine: Boolean? = null,
```

Comment:
Kirill, as you remember, I have a short memory about these solutions, that's why:
1) please write some comment about it - why we have instantiated it with null
2) looks like you have broken the logic for the default value with these arguments, please check

Reply:
1) In process of merging we can met following situation:
Parent config have field `warningTextHasLine = false`
Child config have no value for this field in `toml` config, but since it's have default value `warningTextHasLine = true`,
as the result we will apply this `true`, of which there was no information in config at all.
We can't detect, whether we deal with default value in child config, or it  was provided by user.

2) I think we don't support default values at all, according 1).